<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleShift</class>
 <widget class="QDialog" name="TitleShift">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>161</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Shift titles</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,2,1,2,1">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Move range:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTimeEdit" name="start2">
       <property name="displayFormat">
        <string>hh:mm:ss.zzz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="end2">
       <property name="displayFormat">
        <string>hh:mm:ss.zzz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="maintain2">
       <property name="text">
        <string>Maintain length</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTimeEdit" name="start1">
       <property name="displayFormat">
        <string>hh:mm:ss.zzz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="end1">
       <property name="displayFormat">
        <string>hh:mm:ss.zzz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="maintain1">
       <property name="text">
        <string>Maintain length</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>To range:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TitleShift</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TitleShift</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
